javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -27,14 +28,91 @@
 import org.plumelib.util.ImmutableTypes;
 
 /** A utility class that helps with {@link TypeMirror}s. */
-// TODO: This class needs significant restructuring
 public final class TypesUtils {
 
-    // Class cannot be instantiated
+    /** Class cannot be instantiated */
     private TypesUtils() {
         throw new AssertionError("Class TypesUtils cannot be instantiated.");
     }
 
+    /// Creating types
+
+    // TODO: What order of arguments for typeFromClass is desirable?  Current code is inconsistent,
```

Comment:
Which order of arguments do you prefer?

Reply:
I don't have a preference. It looks like the  `Elements elements, Types types` version was added on 9/18/20,  c1632554dcf1f24d3da6236a677d875299edebbf so I would keep the preexisting one.